package.json

```diff
@@ -4,12 +4,13 @@
   "description": "Arbitration Outsourcing for State Channels",
   "main": "index.js",
   "scripts": {
-    "test": "npx tsc && mocha build/__pisa_tests__ --exit",
+    "test": "npm run build && mocha $(find build/__pisa_tests__/*.js ! -path '*quickTest*') --exit",
```

Comment:
this can be changed back to the way it was now

Reply:
since we have test-docker for testing docker

Reply:
done